From 4264dcfd103debe2d083176e8734e8a0b17e6a93 Mon Sep 17 00:00:00 2001
From: Wang Wen <wangwenpeter@gmail.com>
Date: Tue, 17 Mar 2020 15:27:25 +0800
Subject: [PATCH 5/6] Correct PowerManager transaction IDs. These transaction
 IDs must be kept in sync with the method order from IPowerManager.aidl.

Test: build, compare to java aidl output

Merged-In: I3123a8803c334f7e82fe6da3df0202eb56786b1c
Change-Id: I89fb072b7d2d44f165a7116d993dbdc047827424
Signed-off-by: Wang Wen <wangwen1@meizu.com>
---
 include/powermanager/IPowerManager.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/powermanager/IPowerManager.h b/include/powermanager/IPowerManager.h
index 853f0c9fde..964e318584 100644
--- a/include/powermanager/IPowerManager.h
+++ b/include/powermanager/IPowerManager.h
@@ -46,10 +46,10 @@ public:
         IS_POWER_SAVE_MODE           = IBinder::FIRST_CALL_TRANSACTION + 12,
         GET_POWER_SAVE_STATE         = IBinder::FIRST_CALL_TRANSACTION + 13,
         SET_POWER_SAVE_MODE_ENABLED  = IBinder::FIRST_CALL_TRANSACTION + 14,
-        REBOOT                       = IBinder::FIRST_CALL_TRANSACTION + 17,
-        REBOOT_SAFE_MODE             = IBinder::FIRST_CALL_TRANSACTION + 18,
-        SHUTDOWN                     = IBinder::FIRST_CALL_TRANSACTION + 19,
-        CRASH                        = IBinder::FIRST_CALL_TRANSACTION + 20,
+        REBOOT                       = IBinder::FIRST_CALL_TRANSACTION + 21,
+        REBOOT_SAFE_MODE             = IBinder::FIRST_CALL_TRANSACTION + 22,
+        SHUTDOWN                     = IBinder::FIRST_CALL_TRANSACTION + 23,
+        CRASH                        = IBinder::FIRST_CALL_TRANSACTION + 24,
     };
 
     DECLARE_META_INTERFACE(PowerManager)
-- 
2.20.1

